index.js

```diff
@@ -184,20 +184,24 @@ function colorDelta(img1, img2, k, m, yOnly) {
     let b2 = img2[m + 2];
     let a2 = img2[m + 3];
 
-    if (a1 === a2 && r1 === r2 && g1 === g2 && b1 === b2) return 0;
+    if (a1 === a2 && r1 === r2 && g1 === g2 && b1 === b2 && a1 === a2) return 0;
+
+    const rBackground = 48 + 159 * (k % 2);
+    const gBackground = 48 + 159 * (Math.floor(k / 1.616) % 2);
+    const bBackground = 48 + 159 * (Math.floor(k / 2.612) % 2);
```

Comment:
Hmm, left a comment here but it disappeared somewhere... Can you describe how these specific values / coefficients were picked, and maybe show a screenshot of how this looks?

Reply:
I think one of the tests shows that - for me it looked like how the color picker looks, and I guess the coefficients cane from there...

Reply:
This is what the background looks like:
![background](https://github.com/user-attachments/assets/cba4fa12-482f-43f3-83f5-23d14a961334)

There are a million choices for what the background image could look like. Here are my considerations when creating this background image:

1. The background should not be a uniform color.
2. The background should not contain large areas of uniform color.
3. The background should have a large amount of perceptual variability.
4. The background should be deterministic.
5. The background color should be easy to compute.
6. The background color should be a function of pixel index only (not x,y), because that is what was already available in `colorDelta()`.
7. The background should not contain "common colors" (particularly white and black).
8. The background should not contain lines.
9. The background image should not contain anything we would expect to appear in test images.

Reply:
The `48` and `159` were picked to give large color variability, while avoiding white, black, and fully saturated colors.

The `1.616` and `2.612` were picked to keep the pattern from degenerating into lines at certain integer image widths. I've added some digits to make sure theses coefficients preserve their irrationality for very large images :)